habitat/datasets/rearrange/rearrange_dataset.py

```diff
@@ -21,16 +22,16 @@
 
 @attr.s(auto_attribs=True, kw_only=True)
 class RearrangeEpisode(Episode):
-    art_objs: List[List[Any]]
-    static_objs: List[List[Any]]
-    targets: List[List[Any]]
-    fixed_base: bool
-    art_states: List[Any]
-    nav_mesh_path: str
-    scene_config_path: str
-    allowed_region: List[Any] = []
-    markers: List[Dict[str, Any]] = []
-    force_spawn_pos: List = None
+    ao_states: Dict[
```

Comment:
For readability, I think the comments should be separate from the fields.
```
class RearrangeEpisode(Episode):
   """
   ao_states: <comment about what it is>
   """
   ao_states: Dict[str, Dict[int,float]]
```

It might be useful to create types for readability. Like:
```
InstanceHandle = str
Link = int
State = float
```
so the typing is more explicit.